custom/enikshay/reports/generic.py

```diff
@@ -0,0 +1,92 @@
+from collections import namedtuple
+
+from sqlagg.filters import BETWEEN, IN
+
+from corehq.apps.reports.filters.dates import DatespanFilter
+from corehq.apps.reports.generic import GenericReportView
+from corehq.apps.reports.sqlreport import SqlTabularReport, SqlData
+from corehq.apps.reports.standard import CustomProjectReport, DatespanMixin
+from corehq.apps.reports.util import get_INFilter_bindparams
+from corehq.apps.userreports.util import get_table_name
+from custom.enikshay.reports.filters import EnikshayLocationFilter
+from custom.utils.utils import clean_IN_filter_value
+
+TABLE_ID = 'episode'
+
+ReportConfig = namedtuple('ReportConfig', ['domain', 'locations_id', 'start_date', 'end_date'])
```

Comment:
It wasn't supposed to be a stub but nonetheless, I've changed a name of this variable.